test/e2e/utils/launchVscodeWithMsSqlExt.ts

```diff
@@ -37,5 +37,8 @@ export async function launchVsCodeWithMssqlExtension(): Promise<{
         timeout: 10 * 1000, // 10 seconds
     });
 
+    // wait for 5 seconds for the extension to load
+    await new Promise((resolve) => setTimeout(resolve, 5 * 1000));
```

Comment:
Can we use playwright to determine if the extensions has loaded? Something like checking OE contents or something? Timeouts can be very flaky

Reply:
If it is not possible. It might be helpful to add a prompt that appears when the service is loaded during test runs. This would give us a deterministic way to confirm that the extension has fully initialized before we start executing tests.

@Benjin and @caohai any other ideas?